pkg/kf/testutil/integration.go

```diff
@@ -477,6 +477,52 @@ func RetryPost(
 	}
 }
 
+// RetryGet will send a get request until successful, duration has been reached or context is
+// done. A close function is returned for closing the sub-context.
+func RetryGet(
+	ctx context.Context,
+	t *testing.T,
+	addr string,
+	duration time.Duration,
+	expectedStatusCode int,
+) (*http.Response, func()) {
+	ctx, cancel := context.WithTimeout(ctx, duration)
```

Comment:
I think this should have a t.Helper() call at the top so this function isn't shown as the top of the stack if the test fails

Reply:
good catch, added this to RetryPost as well